File: Thullo.WebApi/Thullo.WebApi/Dtos/Card.cs
using System.Collections.Generic;

namespace Thullo.WebApi.Dtos.Card
{
    public class UpdateDescription
    {
        public int CardId { get; set; }
        public string Description { get; set; }
    }

    public class UpdateTitle
    {
        public int CardId { get; set; }
        public string Title { get; set; }
    }

    public class MoveCardToList
    {
        public int CardId { get; set; }
        public int ListId { get; set; }
    }

    public class UpdateCoverUrl
    {
        public int CardId { get; set; }
        public string CoverUrl { get; set; }
    }

    public class Card
	{
        public int Id { get; set; }
        public string CoverUrl { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public int BoardListId { get; set; }
		public List<Dtos.User.User> Users { get; set; }
		public List<Dtos.CardLabel.CardLabel> Labels { get; set; }
	}
}

File: thullo-react-app/src/models/card.ts
import { File } from "./common";
import { User } from "./user";

export class Card {
    id: number;
    coverUrl: string;
    title: string;
    description: string;
    boardListId: number;
    users: User[] = [];
    labels: CardLabel[] = [];
    attachments: CardAttachment[] = [];
    comments: CardComment[] = [];
}

export class CardLabel {
    id: number;
    name: string;
    color: string;
    boardId: number;
}

export class CardAttachment {
    id: number;
    file: File;
    cardId: number;
    createdDate: Date;
    createdBy: User;
}

export class CardComment {
    id: number;
    text: string;
    cardId: number;
    createdBy: User;
    createdDate: Date;
}